src/libs/httpserver/server-helpers.ts

```diff
@@ -16,7 +16,7 @@ export const addBodyClasses = (HTMLstring: string): string => {
 
   const hasClasses = /<body.*?class=("|')(.*?)("|')/i.exec(HTMLstring)
 
-  if (!hasClasses)
+  if (!hasClasses || hasClasses[2] === undefined)
```

Comment:
We should explain this condition, why are we looking for the 3rd class to be undefined?

Reply:
If you are talking about the `if` check, this is just because we use `hasClasses[2]` two lines later and the `noUncheckedIndexedAccess` rule now enforces this check

If you are talking about the use of `hasClasses[2]`, this is because the regex is formed like this with 3 capturing groups. Maybe we should add a comment with some exemples that match the regex?
Also maybe we should use non-capturing groups instead for the quotes: `<body.*?class=(?:"|')(.*?)(?:"|')`

Reply:
I understand, and I was the one to write the regex if memory serves me right... I guess it proves that the piece of code is obscure without a brief comment explaining what's going on in this condition.